packages/migration-graph-ember/src/utils/discover-ember-service-dependencies.ts

```diff
@@ -32,6 +32,11 @@ export function discoverServiceDependencies(
 ): EmberInferredServiceDependency[] {
   const filePath = join(baseDir, pathToFile);
 
+  // TODO Evaluate if we need service discovery of TS files. Currently this parser doesn't work for typescript source.
+  if (extname(filePath) !== '.js') {
```

Comment:
This should be fine as we can manage state of .ts files in the cli/migrate

Reply:
When I tried ac-hub-web it had a partially migrated in-repo addon and 💥 it broke. So this was my hack to get around that.